quickstart/csharp/dotnetcore/from-microphone/helloworld.sln

```diff
@@ -1,25 +0,0 @@
-
```

Comment:
Please don't delete the .sln file. Carbon automaticly builds the samples using the VS solution file in our CI/CD pipeline. And I think we want users to have VS solutions for all Windows samples, even if you can build on the command line. So we need to find a way to document both options (VS build or command line build). So maybe leave the VS and updat the README.md file to mention both options.